pre-commit

```diff
@@ -130,12 +130,16 @@ for changedfile in $files_to_check; do
     #puppet manifest styleguide compliance
     if [[ "$CHECK_PUPPET_LINT" != "disabled" ]] ; then
         if type puppet-lint >/dev/null 2>&1; then
-            if [[ $(echo "$changedfile" | grep -q '\.*\.pp$' ; echo $?) -eq 0 ]]; then
+            if echo "$changedfile" | grep -iq '\.pp$'; then
+              if [[ 0"$failures" -eq 0 ]]; then
```

Comment:
It's defined already on line 6: https://github.com/drwahl/puppet-git-hooks/blob/master/pre-commit#L6 
So no need to redefine again before loop.